commands/spoiler.js

```diff
@@ -0,0 +1,14 @@
+module.exports.execute = async (client, message, args) => {
+
+	let spoilerText = '||' + args.join(' ') + '||';
+	let author = '**' + message.author.username + '**';
+
+	return await message.channel.send(`${author}\n${spoilerText}`);
```

Comment:
It might be useful to @ the user directly here so that other users can see who sent the message! Let me know what you think.

Reply:
Good point, I'll add that 👍